interface_config.js

```diff
@@ -188,7 +188,14 @@ var interfaceConfig = {
      *
      * Note: this mode is experimental and subject to breakage.
      */
-    AUTO_PIN_LATEST_SCREEN_SHARE: 'remote-only'
+    AUTO_PIN_LATEST_SCREEN_SHARE: 'remote-only',
+
+    /**
+     * Decides whether the 'Report a problem' button gets displayed on the Submit feedback dialog.
```

Comment:
Feels kinda odd to have this toggle when the actual implementation is completely up to the embedder. Doesn't CallStats provide an API for us to reprot this, too?

Reply:
@saghul I kind of miss your point here. How CallStats is related? Why do you think it is odd to have this property? Do you have any suggestions for better implementation?

Reply:
This setting is odd because there is no UI for it. Yet it's in `interfaceConfig`. It relies on the fact that the iframe embedder application does the actual reporting of feedback.

What I was thinking is if there was a way for us to enable it also on meet.jit.si, maybe by sending the feedback through CS.

Reply:
There is UI, we actually add `a` HTML element.

If we want to enable it on meet.jit.si via CS this will be totally different story. We can do this in addition to the implementation in the PR, isn't it? So I still can't understand how CS is related to the PR.

I'm sorry for asking again but do you think we need some changes? If yes, what exactly do you have in mind?